components/logging.js

```diff
@@ -129,7 +129,6 @@ async function fetchLogsFromArchive(
     params: {
       "time_window.start_time.seconds": startTimeSeconds,
       "time_window.end_time.seconds": endTimeSeconds,
-      page_size: 50,
```

Comment:
What error was this causing? Removing this will cause FA to return 300 results

Reply:
Fleet archive was running into an internal error when fetching delivery vehicle results. Removing this allows the server to decide the page size instead.